watcher/tool/watcher/main.go

```diff
@@ -31,13 +48,12 @@ func run() error {
 		return trace.Wrap(err)
 	}
 
-	// grafana might be still starting up, wait for it
-	err = waitForGrafana(context.TODO(), grafanaClient)
+	err = lib.WaitForAPI(context.TODO(), grafanaClient)
```

Comment:
This is only needed to wait for the respective service (Grafana or InfluxDB) to start up initially because watcher running in the same pod usually starts faster.